package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "react-gmp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "styled-components": "^5.3.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "@babel/preset-env": "^7.15.6",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/preset-typescript": "^7.15.0",
+    "@types/compression-webpack-plugin": "^6.0.6",
+    "@types/node": "^16.9.1",
+    "@types/react": "^17.0.21",
+    "@types/react-dom": "^17.0.9",
+    "@types/styled-components": "^5.1.14",
+    "@types/webpack": "^5.28.0",
+    "@types/webpack-dev-server": "^4.1.0",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-styled-components": "^1.13.2",
+    "compression-webpack-plugin": "^9.0.0",
+    "cross-env": "^7.0.3",
+    "html-webpack-plugin": "^5.3.2",
+    "ts-loader": "^9.2.5",
+    "ts-node": "^10.2.1",
+    "tsconfig-paths": "^3.11.0",
+    "typescript": "^4.4.3",
+    "webpack": "^5.52.1",
+    "webpack-cli": "^4.8.0",
+    "webpack-dev-server": "^4.2.1",
+    "webpack-merge": "^5.8.0"
+  },
+  "scripts": {
+    "start": "cross-env NODE_ENV=dev webpack server",
```

Comment:
You should bear in mind that `cross-env` is [no longer developed](https://github.com/kentcdodds/cross-env/issues/257), which may be the reason of its [declined popularity](https://www.npmtrends.com/cross-env-vs-dotenv), so you may opt for [corresponding command-line parameter](https://webpack.js.org/api/cli/#node-env), consider alternative solution or expose [webpack config as a function](https://webpack.js.org/configuration/configuration-types/#exporting-a-function) and pass current [mode as a command line parameter](https://webpack.js.org/configuration/configuration-types/#exporting-a-function).

Reply:
Specified webpack configuration instead.
Probably need to update HW description to use webpack env flag or specify config file in the script

> Have DEV and PROD build commands (use env variables)